csrc/scheduler/pointwise.cpp

```diff
@@ -636,6 +657,12 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams& params) {
     // Don't need to worry about view transformations, just merge reference tv
     // as we normally would.
 
+    std::unordered_map<int, int> rfactor_reorder_map =
+        scheduler_utils::maybeRfactorReorderAsAllocationMap(reference_tv);
```

Comment:
It is OK to recompute here. Because `schedulePointwise` should be less frequently called than `getPointwiseHeuristics`